aiohttp/cookiejar.py

```diff
@@ -37,6 +39,17 @@ def __init__(self, *, unsafe=False, loop=None):
         self._next_expiration = ceil(self._loop.time())
         self._expirations = {}
 
+    def load(self, file_path):
+        f = open(file_path, 'rb')
+        self._cookies = pickle.load(f)
```

Comment:
The line should be wrapped by `with` statement for sake of managed resource usage.